Resources/Prototypes/SimpleStation14/Body/Parts/ipc.yml

```diff
@@ -0,0 +1,186 @@
+- type: entity
+  id: PartIPC
+  parent: BaseItem
+  name: "ipc body part"
+  abstract: true
+  components:
+  - type: Damageable
+    damageContainer: Inorganic
+  - type: BodyPart
+  - type: ContainerContainer
+    containers:
+      bodypart: !type:Container
+        ents: []
+  - type: DynamicPrice
+    price: 100
+
+- type: entity
+  id: TorsoIPC
+  name: "ipc torso"
+  parent: PartIPC
+  components:
+  - type: Sprite
+    netsync: false
```

Comment:
remove netsync and move sprite path to base ipc part to avoid duplicating it on each part